client/disputes/utils.ts

```diff
@@ -45,3 +50,9 @@ export const isDueWithin = ( { dueBy, days }: IsDueWithinProps ): boolean => {
 	const isPastDue = now.isAfter( dueByMoment );
 	return isWithinDays && ! isPastDue;
 };
+
+export const isAwaitingResponse = (
```

Comment:
I looked at that but decided against it for now as it would change the method signature.

Reply:
➕ Should this additional use of `isAwaitingResponse` change be handled in a follow-up PR or within this one?

Reply:
Ah, do you mean `isAwaitingResponse` requires a `Dispute`, but `DisputeStatusChip` would only pass in `status`?

Reply:
We could require only the `status` property, rather than the entire `Dispute` object:


```tsx
interface isAwaitingResponseProps {
	status: DisputeStatus;
}
export const isAwaitingResponse = ( {
	status,
}: isAwaitingResponseProps ): boolean => {
	return disputeAwaitingResponseStatuses.includes( status );
};
```

Reply:
I've updated the method to accept `DisputeStatus | string` and updated `DisputeStatusChip` to use `isAwaitingResponse`